### DISCUSSION TITLE
Footer Menü zentriert / footer menu centered

### DISCUSSION THREAD

**Discussion by Ludinator** [templaza/astroid-framework#355] [General]
Hallo, ich habe in Footer 2 ein horizontales Menü integriert. Wie bekomme ich dieses zentriert? Über die Entwicklerkonsole habe ich mir die Bereiche angeschaut - aber der Footer2 ist für mich nicht eindeutig definiert. In den Rows der Template-Einstellungen finde ich leider auch keine Möglichkeit.  Kann mir jemand einen Tiop geben? Danke Gruß Frank.

-------------

Hello, I have integrated a horizontal menu in footer 2. How do I get this centered? I looked at the areas via the developer console - but the footer2 is not clearly defined for me. Unfortunately, I can't find any option in the rows of the template settings either. Can someone give me a tip? Thanks, greetings Frank.

**Comment by Chacapamac** [comment]
@Ludinator 
Hope it help

I just test that and so far I find that adding the following css on the specific Menu Module work.
```
display: flex;
justify-content: center;
```
![Screen Shot 2023-06-29 at 10 54 45 AM](https://github.com/templaza/astroid-framework/assets/1907276/4f3d57d8-72f5-485b-b99a-91831f983d8f)
